nml/generic.py

```diff
@@ -16,6 +16,26 @@
 # -*- coding: utf-8 -*-
 import sys, os, time
 
+# Enable VT100 sequences on windows consoles
+if os.name == "nt":
+    try:
+        from ctypes import byref, windll
+        from ctypes.wintypes import DWORD, HANDLE
+        kernel32 = windll.kernel32
+        h = kernel32.GetStdHandle(-11) # stdout
+        if h is None or h == HANDLE(-1):
+            raise
```

Comment:
is a bare "raise" when not inside an except block valid python?

Reply:
hmm "If no expressions are present, raise re-raises the last exception that was active in the current scope. If no exception is active in the current scope, a RuntimeError exception is raised indicating that this is an error.", so it does work for what I want, ie early exit

Reply:
Fair enough, bit unusual though... (And bare `except` is frowned upon as well - maybe `except RuntimeError` ? )

Reply:
another option is
```
while True:
    try:
        ...
        if exit_early:
            break
        ...
        break
    finally:
        pass
```

Reply:
hmm no need to try..finally in this case